tests/network/vmi_networking.go

```diff
@@ -116,186 +116,190 @@ var _ = SIGDescribe("[Serial][rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com]
 	}
 
 	Describe("Multiple virtual machines connectivity using bridge binding interface", func() {
-		tests.BeforeAll(func() {
-			tests.BeforeTestCleanup()
-
-			// Prepare inbound and outbound VMI definitions
-
-			// inboundVMI expects implicitly to be added to the pod network
-			inboundVMI = tests.NewRandomVMIWithEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskCirros), "#!/bin/bash\necho 'hello'\n")
+		getVmiWithDefaultBridgeInterface := func() *v1.VirtualMachineInstance {
```

Comment:
This and some more changes here could have been done as a per-requisite to this commit, right?
If that is the case, please provide them in focused commits because it is hard to review here multiple changes at once.

If that is not possible, let me know and I will start diving in deeper.

Reply:
It is possible. Done.